_docs_integrate/migration-from-v6-to-v7.md

```diff
@@ -105,3 +105,6 @@ However, the Connector routes associated with these use cases are not affected b
 - It will be possible to load items not only from truncated references, but also from other references.
   For this reason, the [Load item from reference]({% link _docs_use-cases/use-case-transport-load-item-from-reference.md %}) use case was already added in version 6 and the use case of loading an item from a truncated reference was marked as deprecated.
   It has now been deleted with the update to version 7.
+- Furthermore, the use cases of creating a Device, creating a Device onboarding [Token]({% link _docs_integrate/data-model-overview.md %}#token) and getting the Device onboarding information have been removed.
```

Comment:
Ehm.. This guide is for an integrator, integrators never had access to these usecases b/c they are not used in the connector.